packages/cli/src/commands/releasegold/revoke.ts

```diff
@@ -0,0 +1,24 @@
+import { newCheckBuilder } from '../../utils/checks'
+import { displaySendTx } from '../../utils/cli'
+import { ReleaseGoldCommand } from './release-gold'
+
+export default class Revoke extends ReleaseGoldCommand {
+  static description = 'Revoke the given contract instance'
+
+  static args = []
+
+  static examples = ['revoke --contract 0x5409ED021D9299bf6814279A6A1411A7e866A631']
+
+  async run() {
```

Comment:
Consider adding a confirmation prompt to be extra sure that this isn't run accidentally